src/ducks/payments/PaymentValidation.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from 'react'
+import { ButtonLink } from 'cozy-ui/transpiled/react/Button'
+import { getUrlWebView } from './service'
+import { usePaymentContext } from './PaymentContext'
+
+const PaymentValidation = () => {
+  const { payment, biPayment } = usePaymentContext()
+  const [webViewUrl, setWebViewUrl] = useState('')
+
+  useEffect(() => {
+    const loadUrlWebView = async () => {
+      if (payment.id) {
+        const url = await getUrlWebView(payment.id, biPayment)
+        setWebViewUrl(url)
+      }
+    }
+    loadUrlWebView()
+  }, [biPayment, payment.id])
+
+  return (
+    <div>
+      Validate the payment presenting the webview:
+      <div>
+        <ButtonLink
+          href={webViewUrl}
+          target="_blank"
```

Comment:
Does this functionality will be only available in a web environnement? Or also in Banks mobile? 

Since we open in a new tab, how do we go back to banks?